logs/README.md

```diff
@@ -0,0 +1,122 @@
+# Logs
+
+## Log object
+
+```json
+{
+  "type": "string",
+  "meta": "object",
+  "body": "object",
+  "timestamp": "object"
+}
+```
+
+## **Requests**
+
+|               Route               |                      Description                       |
+| :-------------------------------: | :----------------------------------------------------: |
+| [GET /logs/cache](#get-logscache) | Returns the cache logs and its count of last 24 hours  |
+| [GET /logs/:type](#get-logstype)  | Returns all logs according to the provided type of log |
+
+---
+
+## **GET /logs/cache**
+
+Returns the cache logs and its count of last 24 hours.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Body**  
+  None
+- **Headers**  
+  None
+- **Cookie**  
+  rds-session: `<JWT>`
+
+- **Success Response:**
+
+  - **Code:** 200
+
+    - **Content:**
+
+    **Note: **LOGS_COUNT is a number
+
+    ```json
+    {
+      "message": "Cache Logs returned successfully!",
+      "count": "<LOGS_COUNT>",
```

Comment:
Type annotation missing. I can see the note above, but it defeats the purpose of creating these object interfaces.

Reply:
I wanted to give a placeholder to the count but with JSON formatting, it was highlighted as red (not recognized)
What should be the correct way to mention the type annotation ??
```
"count": "<LOGS_COUNT> => number"
```
OR
Instead of placeholder it can put a value
```
"count": 0
```

Reply:
Made a type annotation instead of random strings
```
"count": "number"
```